src/pages/VideoPlayback/style.module.css

```diff
@@ -0,0 +1,49 @@
+.videoCont {
+  width: 100%;
+}
+
+.videoCont iframe {
+  width: 100%;
+  height: 500px;
```

Comment:
suggestion: If it's not necessary use rem here instead of px

Reply:
it is necessary :grin: